recipes-core/packagegroups/packagegroup-core-boot-openwrt.bb

```diff
@@ -32,6 +32,7 @@ RDEPENDS_${PN} = "\
     base-passwd \
     busybox \
     bridge-utils \
+    tcpdump \
```

Comment:
I can remove it from core-boot if you like but I'm not sure how to create a debug package group and where to add it in.